plugins/autochop.cpp

```diff
@@ -415,6 +378,71 @@ static int get_log_count()
     return valid_count;
 }
 
+static int do_chop_designation(bool chop, bool count_only, int *skipped = nullptr)
```

Comment:
Would you mind moving the function back to its original location so the diff is cleaner? It's hard to see what has changed.

Reply:
done